doc/howtos/software-update-server.rst

```diff
@@ -219,26 +217,24 @@ from the command in the Example is:
     tmp-glibc/deploy/swupd/intel-corei7-64/ostro-image-swupd/
 
   This folder contains both data from intermediate steps and the actual
-  SW Update stream.
-
-  Its location is::
+  SW Update stream located here::
 
     tmp-glibc/deploy/swupd/intel-corei7-64/ostro-image-swupd/www/
 
-  This folder will contain also data related to following builds and it is
-  the one that must be exposed to the device in need of maintenance, through
-  a web server, for example nginx or apache.
+  This folder will also contain data related to subsequent builds and
+  must be exposed through a web server (e.g., ``nginx`` or ``apache``) 
+  to the device running the SW update client for getting its maintentance.
```

Comment:
Over in software-update.rst it says, "The client needs an URL exposing the data stream. This requires setting up a web server [ex: Apache, Nginx] pointing to the “www” directory generated by the SW Update Server. The SW Update server does not act as web server, it merely produces data that the manager of the update stream will have to expose through a separate tool."

If that's correct, we could repeat it here?